final/client/src/pages/cart.tsx

```diff
@@ -19,14 +18,14 @@ const Cart: React.FC<CartProps> = () => {
   const { data, loading, error } = useQuery<GetCartItems>(
     GET_CART_ITEMS
   );
-  
+
   if (loading) return <Loading />;
   if (error) return <p>ERROR: {error.message}</p>;
 
   return (
     <Fragment>
       <Header>My Cart</Header>
-      {!data || !!data && data.cartItems.length === 0 ? (
+      {!data || (!!data && data.cartItems.length === 0) ? (
```

Comment:
```suggestion
      {!data || (data && data.cartItems.length === 0) ? (
```

Reply:
Thanks @lifeiscontent - I'll tweak this a bit.